backend.native/build.gradle

```diff
@@ -16,6 +16,7 @@
 
 buildscript {
    apply from: "$rootDir/gradle/kotlinGradlePlugin.gradle"
+   apply plugin: 'project-report'
```

Comment:
What for?

Reply:
For verbosity and make my life easier :) : modern Kotlin compiler bundling represent complex module interdependences, which might confuse our packaging. This plugin give clear picture of artefact dependencies and resolve conflicts easier.